src/Homeworks/Homework2_Task1/Calc_Utils.cs

```diff
@@ -0,0 +1,22 @@
+namespace Stack_Calculator;
+public static class Calc_Utils {
+    public static bool Operation_Sign(this char value) => value == '-' || value == '+' || value == '/' || value == '*';
```

Comment:
```suggestion
    public static bool Operation_Sign(this char value)
        => value == '-' || value == '+' || value == '/' || value == '*';
```

И правильнее назвать IsOperationSign, это же условная проверка